whelktool/scripts/2018/11/fix-musicMedium.groovy

```diff
@@ -0,0 +1,61 @@
+MUSICMEDIUM_TYPE = 'MusicMedium'
+MUSICMEDIUM_PROPERTY = 'musicMedium'
+
+
+boolean remodelToStructuredValue(term) {
+    def remodelledTermList = []
+
+    //Create musicMedium object and add to list
+    term[MUSICMEDIUM_PROPERTY].each {
+        def map = [:]
+        map << ['@type':MUSICMEDIUM_TYPE]
+        map << ['label':it]
+        remodelledTermList << map
+    }
+    term[MUSICMEDIUM_PROPERTY] = remodelledTermList
+    return true
+
+}
+
+void findAndFixValuesInData(data, obj) {
+    if (obj instanceof List) {
+        obj.each {
+            findAndFixValuesInData(data, it)
+        }
+    }
+    else if (obj instanceof Map) {
+        obj.each { key, value ->
+            checkValueInData(data, obj, key, value)
+        }
+    }
+    else {
```

Comment:
Redundant block?

Reply:
I did, and (as we agreed) removing it is good (that or leaving a comment about "skipping any other case").